lib/themes-panel.coffee

```diff
@@ -299,3 +301,7 @@ class ThemesPanel extends View
   matchPackages: ->
     filterText = @filterEditor.getModel().getText()
     @filterPackageListByText(filterText)
+
+  handleEvents: ->
+    @on 'click', '.sub-section .icon-paintcan', (e) ->
+      _.map(e.currentTarget.nextSibling.children, (child) -> child.hidden = not child.hidden)
```

Comment:
Any reason you're using `underscore-plus`'s `map` function here instead of native `Array.map`?